packages/template-retail-react-app/app/pages/checkout/confirmation.jsx

```diff
@@ -34,32 +34,25 @@ import CartItemVariantImage from '../../components/item-variant/item-image'
 import CartItemVariantName from '../../components/item-variant/item-name'
 import CartItemVariantAttributes from '../../components/item-variant/item-attributes'
 import CartItemVariantPrice from '../../components/item-variant/item-price'
+import {useCurrentCustomer} from '../../hooks/use-current-customer'
 
 const CheckoutConfirmation = () => {
```

Comment:
Confirmation page's Order Summary is kinda broken:
![Google Chrome 2023-03-22 at 11 32 49](https://user-images.githubusercontent.com/847300/227004077-da34c8c4-cc44-4a52-a85b-c7e3c3c4a136.png)

Reply:
good catch! now it is fixed, I added useProducts calls to fetch the product details.